packages/rum-react/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@elastic/apm-rum-react",
+  "version": "0.0.0",
+  "description": "Elastic APM Real User Monitoring for React applications",
+  "author": "Hamid <h.jahtalab@gmail.com>",
+  "homepage": "https://www.elastic.co/guide/en/apm/agent/js-base/current/index.html",
+  "license": "MIT",
+  "main": "dist/lib/index.js",
+  "module": "src/index.js",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/elastic/apm-agent-rum-js.git"
```

Comment:
@jahtalab Lets also add directory field here to find relevant readme when publishing, Missed in the review.